### ISSUE TITLE
private-key.pem file doesn't exist

### ISSUE THREAD

**Issue by Archisman-Mridha** [Archisman-Mridha/terraform-aws-self-managed-k8s#1]
While using the module, we receive this error -

```
Error: Invalid function argument
│ 
│   on .terraform/modules/self_managed_k8s_cluster/6-master-node.tf line 27, in resource "aws_instance" "master_node":
│   27:       private_key = file("${path.module}/outputs/private-key.pem")
│     ├────────────────
│     │ while calling file(path)
│     │ path.module is ".terraform/modules/self_managed_k8s_cluster"
│ 
│ Invalid value for "path" parameter: no file exists at
│ ".terraform/modules/self_managed_k8s_cluster/outputs/private-key.pem"; this function works only with files
│ that are distributed as part of the configuration source code, so if this file will be created by a
│ resource in this configuration you must instead obtain this result from an attribute of that resource.
```

**Comment by Archisman-Mridha** [comment]
This module internally creates an SSH key-pair using the *hashicorp/tls* provider. Then the private key is persisted at *outputs/private-key.pem*. This file is then referred to, while creating the master node and the worker nodes.

> The problem here is that the *outputs/private-key.pem* file is not present initially. Thats'y we receive a validation error from Terraform.

## A temporary workaround
**Go to the folder where this module exists (typically *.terraform/modules/module name you provided*) and create an empty *outputs/private-key.pem* file**

**Comment by Archisman-Mridha** [comment]
This issue is fixed in version 0.1.1
